.github/workflows/ci_macos.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         libt_version: ["2.0.9", "1.2.19"]
         qbt_gui: ["GUI=ON", "GUI=OFF"]
-        qt_version: ["5.15.2", "6.5.0"]
+        qt_version: ["6.5.0"]
         exclude:
           - libt_version: "1.2.19"
             qt_version: "6.5.0"
```

Comment:
This `exclude` can be removed now to allow libtorrent `1.2.19` builds.

Reply:
Since it isn't strictly relevant to goal of this PR let's do it in separate one.

Reply:
Also I would like to raise Qt version used in Ubuntu CI.